.circleci/config.yml

```diff
@@ -956,6 +956,52 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Comp-Update-Reconnect-Abort-4N-2C:
+    triggers:
+      - schedule:
+          cron: "15 23 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/4N_2C/UpdateReconnectAbort
+          config_type: "daily"
+          workflow-name: "GCP-Daily-Services-Comp-Update-Reconnect-Abort-4N-2C"
+          requires:
+            - build-platform-and-services
+          pre-steps:
+            - install-tools
+            - attach_workspace:
+                at: /
+
+  GCP-Daily-Services-Comp-Update-Reconnect-Abort-DisPreUpdate-4N-2C:
+    triggers:
+      - schedule:
+          cron: "30 23 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/4N_2C/UpdateReconnectAbortPreUpdate
```

Comment:
The change should not be here. It should be in regression repo. The result path should match the json description, so it should be `AbortDisPreUpdate`

Reply:
Updated. Regression side json also updated